### PR TITLE
add related class to tree

### PR DESCRIPTION
**Pull request by stubbornfox**

### CODE REVIEW

File: steps/s2_interpret_segments.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def interpret_segments(configuration: Configuration, dataset: Dataset) -> None:
             activation_file = activation_path(f'{image_id}.npz')
             prediction_file = prediction_path(f'{image_id}.npz')
 
-            if exists(activation_file) and exists(prediction_file):
+            if exists(activation_file) or exists(prediction_file):
```
Comment (NiclasvanEyk) [inline steps/s2_interpret_segments.py]: I guess this should be changes, as last time?

File: .idea/workspace.xml [hunk 2]
```diff
@@ -20,10 +20,8 @@
   </component>
   <component name="ChangeListManager">
     <list default="true" id="28996273-d3ad-4646-b594-2d179f640ff4" name="Default Changelist" comment="">
-      <change afterPath="$PROJECT_DIR$/tree_model.pkl" afterDir="false" />
```
Comment (NiclasvanEyk) [inline .idea/workspace.xml:23]: Maybe we should add an entry for the `.idea` folder to the `.gitignore`

File: decision_path_with_concept.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    ]
+
+    # class_nodes = [class_node(class_id, image_id) for class_id, image_id in related_class_ids]
+    # for class_nodex_i in class_nodes:
+    #     label = class_nodex_i['data']['label']
+    #     class_nodex_i['data']['weight'] = label
+
+    edges = []
+
+    # for segment_id, concept_id, weight in zip(local_segment_ids, concept_ids, weights):
```
Comment (NiclasvanEyk) [inline decision_path_with_concept.py]: I think we should just delete all these comments, as they are just leftovers from earlier code, right?

**Review by NiclasvanEyk — CHANGES_REQUESTED** [review]

File: decision_path_with_concept.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+
+def has_concept(weight):
+    if weight:
+        weight = '✅'
+    else:
+        weight = '❌'
+    return weight
+
+def draw_decision_path(image_id, dataset):
```
Comment (NiclasvanEyk) [inline decision_path_with_concept.py]: Is this whole file even needed at all? It looks like nothing from this file is actually used and showing similar birds is also implemented in `decistion_path.py`?

**Comment by stubbornfox** [comment]
@NiclasvanEyk I clean up

**Pull request merged**